### DISCUSSION TITLE
SMTP Sending not working

### DISCUSSION THREAD

**Discussion by jonny07** [supabase/supabase#29703] [Questions] [answered]
I tried 3 smtp providers (gmx, gmail, resend). When I want to authenticate always get the error

Error caught: AuthException(message: {"code":"unexpected_failure","message":"Error sending confirmation email"}, statusCode: 500, errorCode: null)
Error message: Authentication error: {"code":"unexpected_failure","message":"Error sending confirmation email"}

I use flutter in Android Studio. I think it must be a supabase bug. Why is there no button in the dashboard to check SMTP settings, every email program does it. I lost several hours and Im quite frustrated.

**Comment by GaryAustin1** [comment, marked as answer]
You should check the auth logs in the UI, maybe there will be more info on an error back from the providers or other issue.

Also you can rule out your app by sending an email with the UI from the auth users view.   There is a send magic link option for existing users or you can create and invite a new user.

Lots of people using resend.

**Reply by GaryAustin1** [reply]
Did you look in the dashboard logs at the auth log?  That should have more info.

Also as you don't show any example of how you filled out your SMTP settings it is hard to know.  Certainly either the settings can be wrong on the SB side, or you are not doing everything you need to do for setting up the SMTP (like with Resend I believe you have to confirm a domain you own).

Some Resend info:  https://resend.com/docs/send-with-supabase-smtp

**Comment by jonny07** [comment]
Thank you for your fast feedback, Im quite frustrated. I did not find any error in the Auth UI (you mean "logs" in the main menue, right?) I tried to send an invitation email to my email address from the dashboard as you proposed, just get the error: "    Failed to invite user: failed to make invite request: Error sending invite email Auth | Supabase 0%" But no information why not, what is the problem. I tried resend with manual configuration and now even linked resend and supabase, nothing works.

The hint for getting my domain authenticated was good, waiting for approval for this now. 

Thanks again!

 
 

Gesendet: Sonntag, 06. Oktober 2024 um 19:27 Uhr
Von: "GaryAustin1" ***@***.***>
An: "supabase/supabase" ***@***.***>
Cc: "jonny07" ***@***.***>, "Author" ***@***.***>
Betreff: Re: [supabase/supabase] SMTP Sending not working (Discussion #29703)


 

You should check the auth logs in the UI, maybe there will be more info on an error back from the providers or other issue.

Also you can rule out your app by sending an email with the UI from the auth users view. There is a send magic link option for existing users or you can create and invite a new user.

Lots of people using resend.

**Comment by jonny07** [comment]
Thanks a lot for your very quick and good support, adding the DNS for resend did the trick. However I even had to change how resend shows the entries (leave out the "send" for MX) in order to make it work, otherwise resend did not find its own entries. Very strange, but got it working now, thanks again, that was very very big help!
